pkgs/development/python-modules/pydmd/default.nix

```
@@ -6,53 +6,59 @@
 , matplotlib
 , numpy
 , pytestCheckHook
+, pytest-mock
 , pythonOlder
 , scipy
 , ezyrb
 }:
 
-buildPythonPackage rec {
-  pname = "pydmd";
-  version = "0.4.0.post2302";
-  format = "setuptools";
+let
+  self = buildPythonPackage rec {
+    pname = "pydmd";
+    version = "1.0.0";
+    setuptools = true;
 
-  disabled = pythonOlder "3.6";
+    disabled = pythonOlder "3.6";
 
-  src = fetchFromGitHub {
-    owner = "mathLab";
-    repo = "PyDMD";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-EYVmaxwOxje3KVrNbvsjwRqQBD7Rje/JK+qB1F7EqA0=";
-  };
+    src = fetchFromGitHub {
+      owner = "PyDMD";
+      repo = "PyDMD";
+      rev = "refs/tags/v${version}";
+      hash = "sha256-vprvq3sl/eNtu4cqg0A4XV96dzUt0nOtPmfwEv0h+PI=";
+    };
+
```

```suggestion

    build-system = [
      setuptools
    ];
```
add `setuptools` to `build-system`.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

---

Thanks a lot! For some reason I expected `pyproject = true` to fail if there's no appropriate `pyproject.toml`.

---

Yeah, this is actually a standard fallback behavior in PEP.

> (Note that if there is no pyproject.toml file or the build-backend parameter is not defined, then the fall-back behaviour is to use setuptools.)
https://peps.python.org/pep-0518/#build-system-table